charts/ceph-csi-cephfs/templates/provisioner-deployment.yaml

```diff
@@ -158,11 +156,6 @@ spec:
                   fieldPath: spec.nodeName
             - name: CSI_ENDPOINT
               value: "unix:///csi/{{ .Values.provisionerSocketFile }}"
-          securityContext:
-            privileged: true
-            capabilities:
-              add: ["SYS_ADMIN"]
-            allowPrivilegeEscalation: true
```

Comment:
@Madhu-1 just wanted to mention one thing here, eventhough we mention kube as a known to work CO, I am wondering does this have any effect on OCP . Because the `previlege` and `caps` are treated with bit different way in OCP. so if at all we can do a quick test of this PR against OCP that would be good. I dont have solid confirmation that, this will break or work , so sharing my thought .

Reply:
as this is a provisioner pod we don't need this to be privileged?

Reply:
I have tested this with OCP 4.8 everything was working fine when the provisioner pod is not privileged.

Reply:
Cool, if you have tested it against one of the OCP versions, we are good . :+1: